deploy-agent/tox.ini

```diff
@@ -10,6 +11,11 @@ max-line-length=100
 deps=-rrequirements_test.txt
 commands=py.test {posargs}
 
+[testenv:black]
```

Comment:
use `tox -eblack` to run black locally

Reply:
Thanks, looks great.  CI is somewhat self documenting, is there an appropriate place in the teletraan repo docs? If not it can wait.

Reply:
Not sure where the best spot is I see a README pointing to the wiki. So unless you have some guidance perhaps we wait?

Reply:
SGTM